test/pluginSpec.js

```diff
@@ -26,17 +36,69 @@ describe('pluginSpec', () => {
         assetCode: 'XRP'
       }
     })
+    this.plugin._api = fakeApi
 
+    debug('connecting plugin')
     await this.plugin.connect()
+    debug('connected')
```

Comment:
If the testnet is reset, the account won't exist anymore and the tests will break. I guess the best alternative is to set up a local rippled instance as I did for the [integration tests](https://github.com/ripple/ilp-plugin-xrp-paychan/tree/master/test/integration).